### PR TITLE
Update Manage-Hyper-V-integration-services.md

### PR DESCRIPTION
**Pull request by DanieleDeAn**
Added multiple usefull info, corrected zombi links, added info about Windows Servers Hosts, Correct info in tables, added Notes

### CODE REVIEW

**Comment by illfated** [comment]
Looks quite good. May I ask you to add MarkDown default indent marker spacing between the indent markers ( `>` ) and its following text, particularly for your added MD Note blobs? :)

**Comment by DanieleDeAn** [comment]
> Looks quite good. May I ask you to add MarkDown default indent marker spacing between the indent markers ( `>` ) and its following text, particularly for your added MD Note blobs? :)

Many Thanks, i have modified the space, please let me now if it is good ;)

**Comment by illfated** [comment]
👍 Very good, thanks a lot. Looks quite nice to me now. Let me have another quick check, then I will be able to post an approval and suggest a final review from the MS Docs team. :)

File: WindowsServerDocs/virtualization/hyper-v/manage/Manage-Hyper-V-integration-services.md [hunk 1]
```diff
@@ -307,4 +310,7 @@ For hosts earlier than Windows Server 2016 and Windows 10, you'll need to manual
   
 4.  After the installation finishes, all integration services are available for use.
 
-These steps can't be automated or done within a Windows PowerShell session for online virtual machines. You can apply them to offline VHDX images; [see this blog post](https://blogs.technet.microsoft.com/virtualization/2013/04/18/how-to-install-integration-services-when-the-virtual-machine-is-not-running/).
+> [!NOTE]
+> These steps **can't be automated** or done within a Windows PowerShell session for **online** virtual machines.
+> You can apply them to **offline** VHDX images; [see this article](https://docs.microsoft.com/en-us/virtualization/community/team-blog/2013/20130418-how-to-install-integration-services-when-the-virtual-machine-is-not-running).
```
Comment (illfated) [inline WindowsServerDocs/virtualization/hyper-v/manage/Manage-Hyper-V-integration-services.md]: 1. MS Docs recommend using language agnostic links (as long as the link does not break when removing the /en-us/ part of the link)
2. Links are normally labeled with the page title the link points to, unless the title is more or less part of the preceding description (and the title is long enough to become cluttering or less readable inside a note format blob sized text).
```suggestion
> You can apply them to **offline** VHDX images; see [How to install integration services when the virtual machine is not running](https://docs.microsoft.com/virtualization/community/team-blog/2013/20130418-how-to-install-integration-services-when-the-virtual-machine-is-not-running).
```

File: WindowsServerDocs/virtualization/hyper-v/manage/Manage-Hyper-V-integration-services.md [hunk 2]
```diff
@@ -307,4 +310,7 @@ For hosts earlier than Windows Server 2016 and Windows 10, you'll need to manual
...
+> You can also automate the deploy of the integration services through **Configuration Manager** with the VMs **Online**, but you need to restart the VMs at the end of the installation; [see this article](https://docs.microsoft.com/en-us/archive/blogs/manageabilityguys/deploying-hyper-v-integration-services-to-vms-using-config-manager-and-dism)
```
Comment (illfated) [inline WindowsServerDocs/virtualization/hyper-v/manage/Manage-Hyper-V-integration-services.md]:
```suggestion
> You can also automate the deploy of the integration services through **Configuration Manager** with the VMs **Online**, but you need to restart the VMs at the end of the installation; see [Deploying Hyper-V Integration Services to VMs using Config Manager and DISM](https://docs.microsoft.com/archive/blogs/manageabilityguys/deploying-hyper-v-integration-services-to-vms-using-config-manager-and-dism)
```
Please also note: if you disagree with parts of my suggestions, you are free to edit the commit suggestion during the commit process (or you can make the necessary changes yourself to get the desired result).

**Comment by illfated** [comment]
@scooley : Could we have a quick check from you regarding the content and if you would like to suggest any changes?

File: WindowsServerDocs/virtualization/hyper-v/manage/Manage-Hyper-V-integration-services.md (hunk 2 above)
Comment (DanieleDeAn) [reply WindowsServerDocs/virtualization/hyper-v/manage/Manage-Hyper-V-integration-services.md]: Many tnx for the suggestion ;)

**Comment by illfated** [comment]
(ref. #2802 **Link broken** )

**Comment by illfated** [comment]
(ref. #3726 Another broken link)

**Pull request merged**